scripts/mysql-test.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+mysql -h 127.0.0.1 -P 23306 -u root -D gitpod --select-limit=200 --safe-updates --password="test"
```

Comment:
File looks unused, is this leftover?

Reply:
It's a script to get a `mysql` session to the test-db